package.json

```diff
@@ -21,8 +21,13 @@
   },
   "homepage": "https://github.com/learning-as-code/corpus-analysis#readme",
   "dependencies": {
+    "estree-walker": "^3.0.1",
     "git-clone": "^0.2.0",
+    "js-yaml": "^4.1.0",
     "line-reader": "^0.4.0",
```

Comment:
if this isn't used, we can probably remove it

Reply:
it is used.  the report is built as a JS object then `js-yaml` is used [to convert the object to a YAML](https://github.com/learning-as-code/corpus-analysis/blob/680b8760cab262af974fcc0bbd24f74a42883e9a/count-things/index.js#L36) string before writing the `report.yml` files